.changeset/kind-shoes-travel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/form-pill-group': patch
```

Comment:
Since we didn’t merge version packages, the last PR hasn’t released. So we don’t need this changeset I dont think

Reply:
oh! nice

Reply:
To make danger play nice you can just modify the existing change set so that it sees it's part of the changeset of files.